src/profile.ts

```diff
@@ -53,13 +53,17 @@ async function addUserInfo (uok: UsernameOrKaid): Promise<void> {
 
 	getJSON(`${userEndpoint}/profile?${uok.type}=${uok.id}`, {
 		dateJoined: 1,
+		countVideosCompleted: 1,
 		kaid: 1
 	})
 		.then(data => data as UserProfileData)
 		.then(User => {
 			const dateElement = document.querySelectorAll("td")[1];
 			dateElement!.title = formatDate(User.dateJoined);
 
+			const videoCountElement = document.querySelectorAll("td")[5];
+			videoCountElement!.innerText = User.countVideosCompleted.toString();
```

Comment:
I would've used `textContent` instead of `innerText`, since textContent is a property of `Node`, and `innerText` is a property specific to elements. But it doesn't matter.